schema.graphql

```diff
@@ -195,6 +195,18 @@ type GraphNetwork @entity {
   minimumDisputeDeposit: BigInt!
   "Reward to Fisherman on successful disputes. In parts per million"
   fishermanRewardPercentage: Int!
+
+  # Bridge totals (Only available on L1 networks)
+  "Total amount of GRT deposited to the L1 gateway. Note that the actual amount claimed in L2 might be lower due to tickets not redeemed."
+  totalGRTDeposited: BigInt!\
```

Comment:
There's a small typo here that breaks the schema (`\`).

I'm gonna deploy it without it in the meantime, but make sure to remove it ;)

Reply:
wooops thanks for spotting that